packages/web-react/src/components/Tooltip/stories/UncontrolledTooltip.stories.tsx

```diff
@@ -1,53 +1,117 @@
+import React, { useEffect, useRef } from 'react';
+import { Markdown } from '@storybook/blocks';
 import type { Meta, StoryObj } from '@storybook/react';
-import React from 'react';
-import { TooltipWrapper, UncontrolledTooltip } from '..';
+
+import { SpiritTooltipProps } from '../../../types';
 import { Button } from '../..';
-import { Placements } from '../../../constants';
+import ReadMe from '../README.md';
+import { UncontrolledTooltip, TooltipPopover, TooltipTrigger } from '..';
+
+const Placements = [
```

Comment:
Can we get this object somewhere from types/dictionaries?

Reply:
imported from dictionary, but i need to test this more